packages/hoppscotch-sh-admin/src/helpers/auth.ts

```diff
@@ -154,15 +154,7 @@ export const auth = {
   getCurrentUserStream: () => currentUser$,
   getAuthEventsStream: () => authEvents$,
   getCurrentUser: () => currentUser$.value,
-  checkCurrentUser: async () => {
-    try {
-      const res = await authQuery.getUserDetails();
-      return res.data.data.me;
-    } catch (err) {
-      return null;
-    }
-  },
-
```

Comment:
We'd want the errors in this case as well; hence making it point to the `getUserDetails` function above.